### ISSUE TITLE
README.rst is not rendering on PyPI

### ISSUE THREAD

**Issue by theseanything** [cloudreach/sceptre#16]
The README.rst is not rendering on the PyPi site. The PyPi can be quite unforgiving on small errors or warnings when rendering. Running `python setup.py check --restructuredtext -s` shows a few warnings that could be the cause:
```
running check
warning: check: Cannot analyze code. Pygments package not found. (line 130)

warning: check: Duplicate explicit target name: "documentation".

error: Please correct your package.
```

**Comment by theseanything** [comment]
`warning: check: Duplicate explicit target name: "documentation".` can be solved by adding double underscores at the end of `'Documentation <http://sceptre.cloudreach.com/docs/>'__`

**Comment by theseanything** [comment]
#46

**Comment by rmbennett** [comment]
Out of interest - why does this fix it?

**Comment by jamesroutley** [comment]
@rmbennett Looks like it's to do with this: http://stackoverflow.com/a/27435441/4129860

**Comment by jamesroutley** [comment]
Fixed in [v1.1.0](https://pypi.python.org/pypi/sceptre/1.1.0)
